Automattic/wp-calypso
# Staging Sites: Display a module when viewing from the Staging Site

Fixes https://github.com/Automattic/dotcom-forge/issues/1929

## Proposed Changes

Adds a module to 'Hosting Configuration to link the user back to their production site:

<img width="799" alt="image" src="https://user-images.githubusercontent.com/36432/231017166-5bdf6606-6ccc-414a-9b36-e3a3526f65fb.png">

## Testing Instructions

1. Create a new Business site and take it Atomic.
2. Navigate to 'Hosting Configuration' and verify the normal 'Staging site' module appears as expected.
3. Create a staging site.
4. Navigate to 'Hosting Configuration' for the staging site.
5. Verify this new module appears as expected.
